m4_sessions/1811_inning/failure.md

```diff
@@ -25,6 +29,26 @@
 
 ## Session Outline (Agenda for session, include timing for each session and who is facilitating)
 
+Introductions, name, mod, something you are good at and how long it took you to actually become good at it 5 mins
+
+Short video  talking about how failure is a good thing 5 mins
```

Comment:
Please make sure to link the video

Reply:
https://www.youtube.com/watch?v=rBUSsP9rDsw&list=PL4I0udEHucIbGoeBQ30f79fDfcXTrSIIn&index=2 this is our video reframing how failure is a good thing